packages/trader/src/Modules/Trading/Components/Form/ContractType/__tests__/contract-type-display.spec.tsx

```diff
@@ -74,7 +75,7 @@ const list = [
 
 const mocked_props = {
     onClick: jest.fn(),
-    value: 'rise_fall',
+    value: TRADE_TYPES.RISE_FALL as string,
```

Comment:
@kate-deriv doesn't it work without assertion (as string)? :)

Reply:
Will check

Reply:
It was not not working without assertion because of 3d test case, as there I rewrite this field (changed to `value:{TRADE_TYPES.OVER_UNDER}`).
Anyway, I decided to get rid of assertion and pass new value as an independent prop